### ISSUE TITLE
document recommended way to use ocaml-general

### ISSUE THREAD

**Issue by haochenx** [kxcteam/pubinfra.dockerfiles#24]
- ref: https://github.com/kxcdev/prr/blob/9d67d8938bca71ff4b712b020ee13f725ef2dc77/.github/workflows/ci.yml#L10-L45

**Comment by kxc-wraikny** [comment]
@haochenx I don't have correct accessibility to kxcdev/prr in order to push my commit to the repo.

**Comment by haochenx** [comment]
>I don't have correct accessibility to kxcdev/prr in order to push my commit to the repo.

invitation sent.

though this issue is about adding documents to this repository and the link to kxcdev/prr is simply a pointer of usage example that may help writing such documents. (also this issue is kinda low priority now)
fyi: @kxc-wraikny

**Comment by kxc-wraikny** [comment]
@haochenx 
I thought this issue is about changing the container image used by CI in kxcdev/prr to`ghcr.io/kxcteam/ocaml-general:ubuntu.22.04-ocaml.${{ matrix.ocaml-version }}-node.hydrogen-amd64`, is that wrong?

**Comment by haochenx** [comment]
nope. I have changed the title to better reflect my intention (sorry that it wasn't clear before)
@kxc-wraikny
